src/pages/RestaurantDetailPage/RestaurantDetail.test.jsx

```diff
@@ -0,0 +1,24 @@
+import { render } from '@testing-library/react';
+
+import RestaurantDetail from './RestaurantDetail';
+
+describe('RestaurantDetail', () => {
+  const renderComponent = ({ restaurant }) => render(
+    <RestaurantDetail restaurant={restaurant} />,
+  );
```

Comment:
```suggestion
  const renderComponent = ({ restaurant }) => render((
    <RestaurantDetail restaurant={restaurant} />
  ));
```

괄호를 2개로 감싸주시면 콤마를 없애실 수 있어요.